SDPi_Supplement/asciidoc/volume1/use-cases/tf1-ch-c-use-case-sicdsp.adoc

```diff
@@ -0,0 +1,45 @@
+// = Use Case Feature: Standalone ICU Dashboard SP (SICDsp)
+
+[sdpi_offset=3]
+=== Use Case Feature {var_use_case_id}: Standalone ICU Dashboard Single Patient (SICDsp)
+
+==== Narrative:
+
+Dr. Reich is in one of her patient’s ICU room checking on their status. She can view previous radiology results, electrosurgical equipment settings, patient readings such as HR, Blood Pressure, SpO2 and associated waveforms integrated on his real-time ‘Dashboard’ display.  The dashboard display can display visual alarms but does not sound alerts or provide any remote-control capabilities.  (This display can be considered an xDISsp as described in Annex A.)
```

Comment:
"xDISsp as described in Annex A." -> add reference here?

Reply:
anchor added to the 60601-1-8 file (formerly identified as Annex A in the 1.5 use cases document).  Verify that the anchor is properly formed and labeled!